lib/src/presentation/widget/show_img.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+import '../../core/util/constants.dart';
+
+Widget showImgIfExists(String? img, double height) {
+  if (img != null) {
+    return Image.network(
+      img,
+      height: height,
+    );
+  } else {
+    return Image.asset(
+      Constants.backCardImage,
+      height: height,
+    );
+  }
+}
```

Comment:
Instead to do this widget, you can use CachedNetworkImage

Reply:
Done